llrpConnector/README.md

```diff
@@ -0,0 +1,238 @@
+* **debug** - General messages sent/received to/from the RFID Reader
+
+The `loglevel` is hierarchical in that specififying a lower level will send all log levels up to the desired level.  For example, specifying `info` will send `error`, `warn`, and `info` log messages.
+
+The JSON format of a Log message contains the following properties:
+* **eventType** - Either `errorLog`, `warnLog`, `infoLog`, or `debugLog`, depending on the specified `logLevel` in the Vantiq source configuration.
+* **sourceName** - The Vantiq source name which is used to differentiate when multiple RFID Readers are connected to the same LLRP Connector.
+* **hostname** - The RFID Reader's hostname 
+* **readerId** - The RFID Reader's ID
+* **msg** - Log message provided
+
+Log Message Example:
+```
+{   "eventType": "infoLog",
+    "sourceName": "ZebraRFID",
+    "hostname": "fx7500fcc3e9",
+    "readerId": "84248dfffffcc3e9",
+    "msg": "Connection to Reader was successful"
+}
+```
+**Note**: To capture the Reader Capabilities and Reader Configuration, set the `logLevel` to `debug`.  This information can be used when using a new RFID Reader and want to understand the capabilities of the RFID Reader and configuration settings it was set up to use.
+
+### Reader Status Message
+
+Whenever the connection between the LLRP Connector and the RFID Reader changes, a Reader Status message is sent to the Vantiq Source.
+
+The JSON format of Reader Status message contains the following properties:
+* **eventType** - `readerStatus`
+* **readerId** - The RFID Reader's ID
+* **readerOnline** - Boolean value where `True` is `Online` and 'False` is `Offline`
+
+Log Message Example:
+```
+{   "eventType": "readerStatus",
+    "readerId": "84248dfffffcc3e9",
+    "readerOnline": True
+}
+```
+### Reader Antenna Data Message
+
+This message is sent during the initial connection of the LLRP Connector to the RFID Reader.
+
+The JSON format of Reader Antenna Data message contains the following properties:
+* **eventType** - `readerData`
+* **readerId** - The RFID Reader's ID
+* **antennaIds** - An array of the potential antenna IDs possible for the RFID Reader.  While an RFID Reader may support multiple antennas, not all ports will have an actual antenna connected to the RFID Reader.`
+
+Log Message Example:
+```
+{   "eventType": "readerData",
+    "readerId": "84248dfffffcc3e9",
+    "antennaIds": [1,2,3,4]
+}
+```
+
+### RFID Reader Tag Data
+
+The RFID Reader Tag Data is sent to the Vantiq Source as the LLRP Connector receives the data from the RFID Reader.  When the connection is down between the RFID Reader and the LLRP Connector, the RFID Reader caches any Tags found in the antenna's Field Of View (FOV). One reconnected, the cached tags read will be sent to the Vantiq source.
+
+Additionally, based on the configured `tagReadInterval` value specified in the Vantiq source, tag data will be received by the LLRP Connector and Tag Data message is sent to the Vantiq source.

```

Comment:
Yes, exactly.  Basically, you can think of it where the antenna transmits power, the tags charge up and send their information until they can't send any more (in essence they run out of power).  The antenna counts how many times the each tag has sent their information.  Changed 2nd paragraph to state:

RFID Tags will continually send their ID while charged by the RFID Reader through an antenna.  The RFID Reader will 
count the number of times a tag reports since the last time it was asked to report.  The configured `tagReadInterval` 
value specified in the Vantiq source, identifies the frequency the RFID Reader should report its findings from all 
antennas.  This reported findings is received by the LLRP Connector and the RFID Reader Tag Data Message is sent 
to the Vantiq Source.